app/views/exchange_rates/show.html.erb

```diff
@@ -0,0 +1,94 @@
+<% content_for :top_breadcrumbs do %>
+   <%= generate_breadcrumbs "Exchange rates #{@exchange_rates_list.year}",
+                            [ [t('breadcrumb.home'), home_path],
+                            [t('breadcrumb.tools'), tools_path],
+                            [t('breadcrumb.exchange_rate_list'), exchange_rates_list_path], ] %>
+ <% end %>
+
+ <div class="govuk-grid-row">
+   <span class="govuk-caption-xl"><%= trade_tariff_heading %></span>
+   <h1 class="govuk-heading-l"><%= @month_and_year %> monthly exchange rates</h1>
+   <p class="govuk-body-l">Official <%= @month_and_year %> HMRC foreign currency exchange monthly rates for Customs and <abbr title="value added tax">VAT</abbr>.</p>
+
+   <dt class="gem-c-metadata__term">From:</dt>
+   <dd class="gem-c-metadata__definition">
+     <strong><a href="https://www.gov.uk/government/organisations/hm-revenue-customs">HM Revenue &amp; Customs</a></strong>
+   </dd>
+   <dt class="gem-c-metadata__term">Published:</dt>
+   <dd class="gem-c-metadata__definition"><%= @published_date %></dd>
+   <dt class="gem-c-metadata__term">Download:</dt>
+   <dd class="gem-c-metadata__definition">
+     <% files.each do |file| %>
```

Comment:
You can try and use the code here app/views/exchange_rates/_document_detail.html.erb and put it in a view partial so it can be used in both places if possible

Reply:
I think this is different from document detail but i will put in a partial if can be used somewhere else.